### PR TITLE
Bug #129740 fix: Backend>>Tjdashboard>>Give valid error message if us…

### PR DESCRIPTION
**Pull request by shitalkakade24**
…er tries to delete the Dashboard having widgets under it. Currently it gives SQL error

### CODE REVIEW

File: src/components/com_tjdashboard/administrator/languages/en-GB/en-GB.com_tjdashboard.ini [hunk 1]
```diff
@@ -149,4 +149,5 @@ COM_TJDASHBOARD_WIDGET_FORM_DESC_PARAMS = "You can add widget params here"
 ;Widgets List view
 COM_TJDASHBOARD_DASHBOARDS_FILTER_SEARCH_DESC = "Search in widget title.Prefix with ID:to search for a widget ID"
 
-
+;dashboards view
+COM_TJDASHBOARD_DASHBOARDS_DELETE_ERROR_MESSAGE = 'Something Went wrong OR first delete Widgets under this dashboard'
```
Comment (coolbung) [inline src/components/com_tjdashboard/administrator/languages/en-GB/en-GB.com_tjdashboard.ini]: Change this to

Could not delete the dashboard. If this dashboard has any widgets, please delete the widgets before deleting the dashboard

Comment (shitalkakade24) [reply src/components/com_tjdashboard/administrator/languages/en-GB/en-GB.com_tjdashboard.ini]: this show error message if dashboard has any widgets & never delete the dashboard until user not delete the widgets.

Comment (coolbung) [reply src/components/com_tjdashboard/administrator/languages/en-GB/en-GB.com_tjdashboard.ini]: Yes. The message that I proposed reflects that.

Comment (shitalkakade24) [reply src/components/com_tjdashboard/administrator/languages/en-GB/en-GB.com_tjdashboard.ini]: done

File: src/components/com_tjdashboard/administrator/tables/dashboards.php [hunk 2]
```diff
@@ -92,4 +92,30 @@ public function check()
...
+	 * @since   1.0.0
+	 */
+	public function delete($pk = null)
+	{
+		JSession::checkToken() or jexit(JText::_('JINVALID_TOKEN'));
+
+		try
+		{
+			$this->load($pk);
+			$result = parent::delete($pk);
```
Comment (thite-amol) [inline src/components/com_tjdashboard/administrator/tables/dashboards.php]: @shitalkakade24 Update this as per discussion

Comment (shitalkakade24) [reply src/components/com_tjdashboard/administrator/tables/dashboards.php]: done

**Pull request closed without merging**